docs/gestor-dependencias.md

```diff
@@ -0,0 +1,13 @@
+# Gestor de dependencias
+
```

Comment:
En ninguna parte del guión pone "listar los gestores de dependencias y describirlos extensamente y luego elegir uno por razones que no se han mencionado antes".
Se pide que primero entendáis los estándares, buenas prácticas y criterios que afectan a la elección, y *luego* pongáis los que sigan esos estándares y contéis cómo cumplen esos criterios y en qué grado. por favor, tratad de entender cual es *el objetivo de aprendizaje*, para entender qué es lo que hay que hacer (y no lo que hay que escribir)

Reply:
El comentario está puesto en un espacio en blanco, la línea 2. Sólo con borrar ese espacio en blanco se eliminan estos comentarios (si es que ya has hecho lo que se pedía)